src/shared/components/button/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import classNames from 'classnames';
+import Link from 'shared/components/link';
+
+import styles from './index.module.css';
+
+const Button = ({ text, path, customClass }) => (
+    <Link to={ path } className={ styles.link } >
+        <div className={ classNames(styles.customButton, customClass) }>
+            { text }
+        </div>
+    </Link>
+);
+
+Button.propTypes = {
+    text: PropTypes.string.isRequired,
+    path: PropTypes.string,
```

Comment:
I think that this path should be required, since you do not have an additional behavior, such as an `onClick` prop.

Reply:
Agree.

Reply:
Fixed.